test/python/gemm/test_gelu_mlp.py

```
@@ -0,0 +1,387 @@
+def _ref(x, w1, b1, w2, b2):
+    hidden = F.gelu(F.linear(x, w1, b1), approximate="tanh")
+    return F.linear(hidden, w2, b2)
+
+
+def _rel_l2(actual, expected):
+    return (actual.float() - expected.float()).norm().item() / max(expected.float().norm().item(), 1e-9)
+
+
+def _inputs(*, requires=(False, False, False, False, False)):
+    torch.manual_seed(0)
+    M, H, intermediate, O = 128, 256, 512, 192
```

Ruff (0.16.1)

[error] Ambiguous variable name: `O`

(E741)